### PR TITLE
Update flash image

### PR DESCRIPTION
**Pull request by DanielHo-BS**
flash_image/images
flash_image/flash_mfi.rst
flash_image/index.rst
flash_image/recovery_mode.rst

### CODE REVIEW

File: flash_image/flash_mfi.rst [hunk 1]
```diff
@@ -3,4 +3,70 @@
...
+
+.. code-block:: bash
+
+  $> sudo apt install python qemu-user-static
+
+iii) Check ROScube is in recovery mode now.
+
+.. code-block:: bash
+
+  $> lsusb | grep NVIDIA
```
Comment (evshary) [inline flash_image/flash_mfi.rst]: You can remove "$>" for users to copy easily.

File: flash_image/flash_mfi.rst [hunk 2]
```diff
@@ -3,4 +3,70 @@
...
 
-TODO
+1. Parepare mfi image on your host computer.
+--------------------------------------------
+
+i) Download mfi image.  
+
+ a.   Go to the `device web <https://www.adlinktech.com/Products/ROS2_Solution/ROS2_Controller/ROScube-X?lang=en/>`_.  
+ b.  Login or create an account.  
+ c. Click BSP to download the BSP MFI Binary
```
Comment (evshary) [inline flash_image/flash_mfi.rst]: Add the picture to show where is BSP in the website.

File: flash_image/flash_mfi.rst [hunk 3]
```diff
@@ -3,4 +3,70 @@
...
+
+ a.   Go to the `device web <https://www.adlinktech.com/Products/ROS2_Solution/ROS2_Controller/ROScube-X?lang=en/>`_.  
+ b.  Login or create an account.  
+ c. Click BSP to download the BSP MFI Binary  
+
+ii) Install required tools.
+
+.. code-block:: bash
+
+  $> sudo apt install python qemu-user-static
```
Comment (evshary) [inline flash_image/flash_mfi.rst]: qemu-user-static is not necessary.

File: flash_image/index.rst [hunk 4]
```diff
@@ -3,7 +3,13 @@
 How to flash image
 ##################
 
-TODO
+Prerequisite
```
Comment (evshary) [inline flash_image/index.rst]: Don't make Prerequisite be the headline.

File: flash_image/recovery_mode.rst [hunk 5]
```diff
@@ -3,4 +3,52 @@
...
+iii) Release RESET button dirst and then release RECOVERY button.
+
+.. image:: images/release-reset-button.png
+  :width: 80%
+  :align: center
+
+4. Now ROScube is in recovery mode now.
+---------------------------------------
+
+.. image:: images/nvidia-usb.png
```
Comment (evshary) [inline flash_image/recovery_mode.rst:51]: Your image should not contain "ROS 1 noetic"

**Review by evshary — CHANGES_REQUESTED** [review]

**Comment by DanielHo-BS** [comment]
I made new update.
Please check out.

File: flash_image/flash_mfi.rst (hunk 1 above)
Comment (DanielHo-BS) [reply flash_image/flash_mfi.rst]: Remove done.

File: flash_image/flash_mfi.rst (hunk 2 above)
Comment (DanielHo-BS) [reply flash_image/flash_mfi.rst]: Add two pictures.

File: flash_image/flash_mfi.rst (hunk 3 above)
Comment (DanielHo-BS) [reply flash_image/flash_mfi.rst]: remove done.

File: flash_image/index.rst (hunk 4 above)
Comment (DanielHo-BS) [reply flash_image/index.rst]: OK.

File: flash_image/recovery_mode.rst (hunk 5 above)
Comment (DanielHo-BS) [reply flash_image/recovery_mode.rst:51]: I update a new image.

**Comment by evshary** [comment]
LGTM

**Pull request merged**